IBPSA/Fluid/HeatPumps/ModularReversible/BaseClasses/PartialTableData2D.mo

```diff
@@ -0,0 +1,155 @@
+within IBPSA.Fluid.HeatPumps.ModularReversible.BaseClasses;
+partial model PartialTableData2D
+  "Reversible heat pump based on 2D manufacturer data"
+  extends IBPSA.Fluid.HeatPumps.ModularReversible.Modular(
+    final use_rev=true,
+    QCoo_flow_nominal=refCyc.refCycHeaPumCoo.QCooNoSca_flow_nominal*scaFacHea,
+    dpEva_nominal=datTabHea.dpEva_nominal*scaFacHea^2,
+    dpCon_nominal=datTabHea.dpCon_nominal*scaFacHea^2,
+    redeclare replaceable
+      IBPSA.Fluid.HeatPumps.ModularReversible.Controls.Safety.Data.Wuellhorst2021
+      safCtrPar constrainedby
+      IBPSA.Fluid.HeatPumps.ModularReversible.Controls.Safety.Data.Generic(
+      final tabUppHea=datTabHea.tabUppBou,
+      final tabLowCoo=datTabCoo.tabLowBou,
+      final use_TConOutHea=datTabHea.use_TConOutForOpeEnv,
+      final use_TEvaOutHea=datTabHea.use_TEvaOutForOpeEnv,
+      final use_TConOutCoo=datTabCoo.use_TConOutForOpeEnv,
+      final use_TEvaOutCoo=datTabCoo.use_TEvaOutForOpeEnv),
+    dTEva_nominal=(QHea_flow_nominal - PEle_nominal)/cpEva/mEva_flow_nominal,
+    mEva_flow_nominal=datTabHea.mEva_flow_nominal*scaFacHea,
+    mCon_flow_nominal=datTabHea.mCon_flow_nominal*scaFacHea,
+    dTCon_nominal=QHea_flow_nominal/cpCon/mCon_flow_nominal,
```

Comment:
1. I do not suggest adding a Boolean parameter to switch between ∆T- or flow-based parameterization. I suggest to simply remove the ∆T-based parameterization that creates confusion here, and little benefit elsewhere. 
2. Not sure that I follow. My concern is that 
    - the model `Modular` and the models `RefrigerantCycleHeatPumpHeating` and `RefrigerantCycleHeatPumpCooling` inside this model do not have the same value of `m*_flow_nominal` and,
    - the check that `m*_flow_nominal` is within bounds is done based on the values from `RefrigerantCycleHeatPumpHeating` and `RefrigerantCycleHeatPumpCooling` that are not directly provided by the user.
3. `IBPSA.Utilities.Diagnostics.AssertInequality` probably provides the best formulation to avoid any overhead when checking for flow within bounds during simulation. I suspect that the impact is minor, but probably worth checking indeed.

Reply:
1. I found the dT method to be way easier for new users, so I will keep it.
2. Ah yes, I think I found the error: I forgot the set the correct mass flow rates in the modular example, fixed now. Now, warning do occur. The pre-configured models don't have this issue.
3. Those could be an option. However, I think the check during initialization is sufficient for now. Other IBPSA models don't use this, at least it is not used in IBPSA.